plasmapy/physics/parameters.py

```diff
@@ -1452,8 +1480,10 @@ def upper_hybrid_frequency(B, n_e):
         If either of B or n_e contains invalid values or are of
```

Comment:
It'd be good to put backticks around B and n_e here so that they get formatted as code.